pandas/core/indexes/datetimes.py

```diff
@@ -1968,14 +1974,41 @@ def tz_localize(self, tz, ambiguous='raise', errors='raise'):
             .. deprecated:: 0.15.0
                Attempt to infer fall dst-transition hours based on order
 
+
         Returns
         -------
-        localized : DatetimeIndex
+        DatetimeIndex
+            It returns naive time or aware time
```

Comment:
I'd describe it more in terms of the functionality than the types. Something like "Index converted to the specified time zone." (The "It returns" seems redundant, and I think the spec requires a period at the end)

Reply:
I copied your line "index converted to the specified time zone". because It gives perfect means